homeassistant/components/notify/smtp.py

```diff
@@ -179,9 +180,19 @@ def _build_multipart_msg(message, images):
         body_text.append('<img src="cid:{}"><br>'.format(cid))
         try:
             with open(atch_name, 'rb') as attachment_file:
-                attachment = MIMEImage(attachment_file.read())
-                msg.attach(attachment)
-                attachment.add_header('Content-ID', '<{}>'.format(cid))
+                file_bytes = attachment_file.read()
+                try:
+                    attachment = MIMEImage(file_bytes)
+                    msg.attach(attachment)
+                    attachment.add_header('Content-ID', '<{}>'.format(cid))
```

Comment:
Oh yes in the code, duh. File attachments didn't seem to work with that header, so I separated it out into two separate flows